recipes-kernel/linux/kernel-devsrc.bbappend

```diff
@@ -1 +1,7 @@
 require kernel-devsrc-nilrt.inc
+
+do_configure[depends] += "virtual/kernel:do_deploy"
```

Comment:
I don't remember why it was needed for us but not upstream.. maybe its only required for alternate kernels? Can you check if this is required for this recipe?

Reply:
Can you share how I can check that? Just by building kernel-devsrc? I currently have everything built. Should I clean `linux-nilrt` before that?

Reply:
Removing the line and building `kernel-devsrc` should validate it - yes I think `linux-nilrt` should be cleaned before that.

Reply:
Seems to work without this line.